besom-json/src/main/scala/besom/json/ProductFormats.scala

```diff
@@ -19,14 +19,44 @@ package besom.json
 trait ProductFormats:
   self: StandardFormats with AdditionalFormats =>
 
-  def writeNulls: Boolean = false
+  def writeNulls: Boolean             = false
+  def requireNullsForOptions: Boolean = false
 
   inline def jsonFormatN[T <: Product]: RootJsonFormat[T] = ${ ProductFormatsMacro.jsonFormatImpl[T]('self) }
 
 object ProductFormatsMacro:
   import scala.deriving.*
   import scala.quoted.*
 
+  private def findDefaultParams[T](using quotes: Quotes, tpe: Type[T]): Expr[Map[String, Any]] =
```

Comment:
@lbialy correct me if I'm wrong, but this change looks like a rebase/merge issue?

Reply:
NVMD, fixed, wrong base